Add section for contacts

FredsephT32 opened the pull request:
Created this new branch to add the new section of "Contact me", I already added the background image the title, and the paragraph.

FredsephT32 committed:
Add section for contacts

Added H2 tag and p tag.

FredsephT32 committed:
Stylelint Report

Lines 312, 325 fixed

shakerAbuDrais committed:
Add form inputs and the textarea

FredsephT32 committed:
Stylized like figma

The inputs and the textarea is now looking like the figma template.

FredsephT32 committed:
Stylelint Report

Line 330 color minor error solved

mgmediaweb commented on styles.css:
```diff
@@ -263,3 +265,72 @@ button:hover {
+  margin: 0;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
```
Kindly, I suggest you use a thinner weight for the font in this section to make it more similar to the original Figma design.

FredsephT32 replied:
Hello @mgmediaweb 👋
The problem with the weight wasn't just in the "Contact me" section, it was actually on all of them. But thanks! I changed the font-weight directly from the **@import url** itself.

mgmediaweb reviewed (changes requested):
Hi @FredsephT32, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![almost5](https://user-images.githubusercontent.com/41912912/186311210-8fd479e8-3098-4eb5-a08b-4afd57267916.gif)

## Highlights 💯 
- Professional README file. ✔️
- PR has a good title and summary. ✔️
- Good HTML structure. ✔️

## Required Changes ♻️

*** Kindly, I suggest you apply some kind of interaction when using the buttons (normal and hover) original design of Figma.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FredsephT32 committed:
Requested Changes

Like the Reviewer @mgmediaweb said, I changed the weight for the font to be more like in the Figma design.

franclobo commented on styles.css:
```diff
@@ -216,7 +217,9 @@ button:hover {
 .project-check button {
+  font-family: 'Poppins', sans-serif;
   background: transparent;
+  background-color: white;
```
- [x] Please 🙏🏽 implement the button interactions (enable, hover, etc.).

- The last reviewer suggested 👀 :

> Kindly, I suggest you apply some kind of interaction when using the buttons (normal and hover) original design of Figma.

![buttons](https://user-images.githubusercontent.com/58642949/186332950-39239aa8-9962-416a-b54e-8aebd8efcd84.png)

franclobo reviewed (changes requested):
Hi, @FredsephT32 and @shakerAbuDrais, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] **[OPTIONAL]** This is a pair programming activity, each team member should have the same quantity of comments.
- [x] **[OPTIONAL]** Add your partner to the authors' section on the README file.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FredsephT32 committed:
Review button changes

buttons interactions integrated

FredsephT32 committed:
[OPTIONAL] README file

Suggestion from @franclobo to add my partner to the authors' section on the README file.

FredsephT32 commented:
The last important integration was the interaction when using the buttons 👍 

It is too late in my country, I really need to go to sleep😢 bye bye 👋

prolajumokeoni commented:
Hi @FredsephT32 the button interactions have not been implemented, kindly fix this as this is part of the requirements and it is a requested change by previous reviewer 👍🏿

prolajumokeoni commented on index.html:
```diff
@@ -197,5 +197,33 @@ <h3>Skills</h3> <img src="Images/Arrow_right.svg" alt="Arrow_right">
+        
+        <textarea  class="contact-boxes" cols="30" rows="6" id="contact-msg" name="user_message" maxlength="500" placeholder="Write your message here" required></textarea>
+       
+        <div class="project-check"> <button type="submit">Get it touch</button></div>
```
- [x]  Kindly make sure that Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are implemented as shown in the Figma template. please fix this as it was requested by previous reviewer 👍🏿 
- Check the screenshot below:


 ![image](https://user-images.githubusercontent.com/71874425/186142037-41a9e629-218f-4ab5-9f54-cf90ae740945.png)

prolajumokeoni reviewed (changes requested):
Hi @Team
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR title and Description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly ensure that as partners you always have balanced commits in a pair programming project 👍 
- Branch name should have a name of a feature or action related to your project eg. “hello-microverse” or “setup-project”



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FredsephT32 committed:
Extra Requested Changes

button interactions such as hover, active (when it's pressed), disabled, and enable

FredsephT32 committed:
Stylelint report solved

deleted the comments in css.

leonardodiasb reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @FredsephT32 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️ Good PR title and Description

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

FredsephT32 commented:
To anyone that is reading this!😉

Have a great day and please please approve my work.
